tests/client/json_api_client_test.py

```diff
@@ -12,32 +12,29 @@
 
 class JsonApiClientTest(unittest.TestCase):
     def test_pack_params(self):
-        self.assertDictEqual(JsonApiClient._pack_params({
+        self.assertItemsEqual(JsonApiClient._pack_params({
```

Comment:
Perhaps wrap both arguments to this assertion with `set()` since the order is just an implementation detail?